RPIPs/RPIP-60.md

```diff
@@ -54,7 +54,7 @@ The explanation document is a mechanism intended to help the pDAO hold the secur
 **Upgrade Delay Setting**  
 Due to the considerations around slowing down hotfixes [described above](#security-considerations), the initial upgrade delay setting is attempting to be the smallest that allows "enough" time for participants to exit the system ahead of a proposal if they deem that appropriate.
```

Comment:
Is this even still true with the setting to 1 week?

Reply:
I think so -- the bit immediately following explains why I think it's ~true